### DISCUSSION TITLE
Add the ability to hide or only show watched items in a collections and/or playlist

### DISCUSSION THREAD

**Discussion by eyarea51** [advplyr/audiobookshelf#3383] [Ideas]
It would be handy to be able to see the items I have left to read in a collections and/or playlist as well as the reverse.  It helps when I am searching for things I have read or going to be read.

**Comment by djcprone** [comment]
I second this. Filtering with hide completed would be awesome. That said, as far as I can tell in collections, you can't even see what's completed at all so that's the first thing we need, lol. It only shows a check mark for downloaded to local next to the title but nothing to designate it has been read.
